bdio-tinkerpop/src/main/java/com/blackducksoftware/bdio2/tinkerpop/sqlg/SqlgBlackDuckIoReader.java

```diff
@@ -187,7 +187,10 @@ public SqlgNodeAccumulator addNode(Map<String, Object> node) {
                 node.keySet().forEach(k -> schema.put(k, null));
                 Object outVertexId = node.get(JsonLdConsts.ID);
                 identifierTypes.computeIfAbsent(outVertexLabel, SqlgBlackDuckIoReader::newIdentifierSet).put(outVertexId);
-                getNodeEdges(node, (edgeLabel, inVertexId) -> edges.put(EdgeKey.of(edgeLabel, outVertexLabel), Pair.of(outVertexId, inVertexId)));
+                getNodeEdges(node, (edgeLabel, inVertexId) -> {
```

Comment:
Given we have very little knowledge of testing in tinkerpop, are there tests we can put into place to exercise this.   Additionally, can you give more context in regards to this change?

Reply:
This map node (Map<String, Object> node) contains list of all file identification for component.
Earlier single file is getting mapped to component i.e single key value pair is created for all file identifications and rest file is ignored. Now I change to create separate key value pair for all the file identification for component so that all file identification are getting mapped to that component.

Reply:
Test case will take some time as it is not running on local.  Trying to setup db and docker to run test cases. Once I am able to run existing test cases then will add new one.